mbed/mbed_cdc.py

```diff
@@ -0,0 +1,111 @@
+
+def mbed_cdc(port, reset=False, sterm=False, baudrate=9600, timeout= 10, print_term_header=True):
+    try:
+        from serial import Serial, SerialException
+        import serial.tools.miniterm as miniterm
+    except (IOError, ImportError, OSError):
+        return False
+
+    def get_instance(*args, **kwargs):
+        try:
+            serial_port = Serial(*args, **kwargs)
+            serial_port.flush()
+        except Exception as e:
+            error("Unable to open serial port connection to \"%s\"" % port)
+            return False
+        return serial_port
+
+    def cdc_reset(serial_instance):
+        try:
+            serial_instance.sendBreak()
+        except:
+            try:
+                serial_instance.setBreak(False) # For Linux the following setBreak() is needed to release the reset signal on the target mcu.
+            except:
+                result = False
+
+    def cdc_term(serial_instance):
```

Comment:
I'm pretty sure that this function does not capture any state either. Could you move this function out of `mbed_cdc`?

Reply:
It uses other variables from the wrapper function, e.g. imports and cdc_reset()